src/components/cylc/cylcObject/Menu.vue

```diff
@@ -344,14 +321,34 @@ export default {
 
     /* Call a mutation using only the tokens for args. */
     callMutationFromContext (mutation) {
+      this.showMenu = false
       // eslint-disable-next-line no-console
       console.debug(`mutation: ${mutation._title} ${this.node.id}`)
-      mutate(
-        mutation,
-        getMutationArgsFromTokens(mutation, this.node.tokens),
-        this.$workflowService.apolloClient
-      )
-      this.showMenu = false
+
+      if (mutation.name === 'log') {
+        this.$router.push({
+          name: 'workspace',
+          params: {
+            workflowName: this.node.tokens.workflow
+          }
```

Comment:
An explanatory comment linking back to this PR would be good (context easily forgotten).